packages/starlight/vitest.config.ts

```diff
@@ -18,12 +18,14 @@ export default defineConfig({
 				'props.ts',
 				// Main integration entrypoint — don’t think we’re able to test this directly currently.
 				'index.ts',
+				// Since Vitest 2.1.2, coverage is collected for `*.astro` files.
+				'**/*.astro',
 			],
 			thresholds: {
-				lines: 88,
+				lines: 87,
 				functions: 90,
 				branches: 90,
-				statements: 88,
+				statements: 87,
```

Comment:
Since [Vitest 2.0.0](https://github.com/vitest-dev/vitest/releases/tag/v2.0.0), empty lines are now ignored in coverage by default which can cause significant differences in coverage results. I reduced the thresholds only to adjust for this change.